### ISSUE TITLE
demo with multi images 

### ISSUE THREAD

**Issue by zqdeepbluesky** [matterport/Mask_RCNN#775]
hi,thansk for your code.I have a question about demo.py,If I have a folder,and there are a lot of pictures under this folder and I want to detect all of them, but One by one picture to detect a bit slow. so can you show me how to do it? thanks so much.

**Comment by JoeLogan1981** [comment]
Are you talking about inference here, rather than training?

**Comment by zqdeepbluesky** [comment]
Yes, because I have downloaded the coco weight file, so I want to run demo.py to detect all the images in my folder. which code should I modify it, can you help me? thank you very much

**Comment by artisvirat** [comment]
Do not modify anything, the original code contains script to run random pictures in a folder for inference. Just change the path of that folder to yours.

**Comment by zqdeepbluesky** [comment]
I know what you mean, but I want to use demo.py to batch process these images, that is, run a demo.py to detect all the images in the folder,Instead of running demo.py once, you can only randomly detect an image.thanks so much.

**Comment by artisvirat** [comment]
Try a loop or vectorize it
